forte/processors/base/data_augment_processor.py

```diff
@@ -94,23 +618,40 @@ def default_configs(cls):
             - augment_entries: defines the entries the processor
             will augment. It should be a full path to the entry class.
```

Comment:
Note a `full path`, class names are not paths. You should say `full qualified names`. And please add an example in the docstring here.